### PR TITLE
Web 6 contributors section update

### PR DESCRIPTION
**Pull request by renaldox**
# Describe your changes
updating branch with all changes
## Issue ticket number and link
Issue #19 
## Checklist before requesting a review

- [x ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by westerandr** [comment]
Uso can you pull latest changes from main please? and include your component with the rest in page.svelte

**Comment by westerandr** [comment]
But overall looks good! Might need you to work on different colors for dark  and light mode for the cards. Looks good on mobile, might need to reduce space between cards. Good stuff!

**Comment by Green-Ranger11** [comment]
Also don't forget to import and showcase your changes in the `+page.svelte` file!

**Comment by westerandr** [comment]
Also don't forget to reference the issue number in your PR description.

File: src/lib/Components/Contributors.svelte
Comment (Green-Ranger11) [inline src/lib/Components/Contributors.svelte:1]: Hol up we gotta put this in `/components` lowercase please haha

**Review by Green-Ranger11 — CHANGES_REQUESTED** [review]
Just one small change and should be good to go!

**Review by Green-Ranger11 — CHANGES_REQUESTED** [review]
Just one small change and should be good to go!

**Review by Green-Ranger11 — APPROVED** [review]
LGTM! Malo lava!

**Pull request merged**